te/TE_WRAP.py

```diff
@@ -394,7 +397,10 @@ def __buildDocker(type, image_id=None):
         logging.getLogger("paramiko").setLevel(logging.ERROR)
         ssh = paramiko.SSHClient()
         ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        ssh.connect(host, username=uname, password=pwd)
+        if pwd:
```

Comment:
Does this logic take care of no-access ( when no ssh related stuff is provided )

Reply:
The function is to deploy TE Controller in write access. One would not use TE_WRAP to deploy controller, but rather go via docker run command